raiden-dapp/src/components/ConnectDialog.vue

```diff
@@ -78,9 +79,8 @@ export default class ConnectDialog extends Vue {
   close() {}
 
   @Emit()
-  connect(subkey?: true) {
-    this.hideClose = true;
-    return subkey;
+  connect(uploadedState: string, subkey?: true) {
+    return { uploadedState, subkey };
```

Comment:
Is there any specific reason why we don't return the `undefined` here if empty instead of https://github.com/raiden-network/light-client/pull/1308/files#diff-8f4f0c339d26d1e6c6c13add936452d9R91